mkosi

```diff
@@ -1420,6 +1420,11 @@ def prepare_tree(args: CommandLineArguments, root: str, do_run_build_script: boo
             with open(os.path.join(dracut_dir, "30-mkosi-uefi-stub.conf"), "w") as f:
                 f.write("uefi_stub=/usr/lib/systemd/boot/efi/linuxx64.efi.stub\n")
 
+        if args.bootable and args.distribution == Distribution.mageia:
+            with open(os.path.join(dracut_dir, "51-mkosi-override-mageia.conf"), "w") as f:
+                f.write("hostonly=no\n")
```

Comment:
Last one found applies right? Mageia has 50-mageia.conf in /etc/dracut.conf.d that sets hostonly=yes so we override it again to no

Reply:
This is a terrible case of whack-a-mole, but maybe promoting `30-mkosi-hostonly` to `55-mkosi-hostonly` would leave us with a single file doing one thing, but I fear at some point this might not be resolvable for all distros anymore.

Reply:
We have to add a separate file anyway for to override omit_dracutmodules again. Might as well to hostonly there as well.